src/components/moderation/modmail.ts

```diff
@@ -40,6 +44,29 @@ export default class Modmail extends BotComponent {
         component_state: moderation_state;
     }>();
 
+    override async setup(commands: CommandSetBuilder) {
+        commands.add(
+            new TextBasedCommandBuilder("wsetupmodmailsystem", EarlyReplyMode.none)
```

Comment:
It would be good to make this a non-slash command so it's harder to accidentally run from a bad tab completion

Reply:
Same as in the other PR: I thought they are non-slash unless .set_slash(true)`?

Reply:
Cross-posting from other PR: Slash is default, I'll add .set_slash(false)